docs_user/modules/con_adoption-prerequisites.adoc

```diff
@@ -37,6 +37,10 @@ Tools::
 * Install the `oc` command line tool on your workstation.
 * Install the `podman` command line tool on your workstation.
 
+{OpenStackShort} {rhos_prev_ver} release::
+
+* The {OpenStackShort} {rhos_prev_ver} cloud is updated to the latest patch release.
```

Comment:
```suggestion
* The {OpenStackShort} {rhos_prev_ver} cloud is updated to the latest minor version of the {rhos_prev_ver} release.
```
I think we try to avoid saying "patch" in downstream docs.